e2e/cypress/integration/api/iam/policies_api.spec.ts

```diff
@@ -1,22 +1,47 @@
+const defaultAdminReq = {
+  headers: {}, // must fill in before use
+  url: '/apis/iam/v2/policies'
+};
+const cypressPrefix = 'test-policies-api';
+const now = Cypress.moment().format('MMDDYYhhmm');
+const project1 = {
+    id: `${cypressPrefix}-project1-${now}`,
+    name: 'Test Project 1'
+};
+const project2 = {
+    id: `${cypressPrefix}-project2-${now}`,
+    name: 'Test Project 2'
+};
+const policyID = `${cypressPrefix}-policy-${now}`;
+const policyName = `${cypressPrefix} policy ${now}`;
+
+const chefManagedPolicyIDs = [
+  'administrator-access',
+  'editor-access',
+  'viewer-access',
+  'ingest-access'
+];
+
+interface Policy {
+  id: string;
+  name: string;
+  members: string[];
+  statements: Statement[];
+  projects: string[];
+}
+
+interface Statement {
+  effect: string;
+  actions: string[];
+  projects: string[];
+}
+
+
```

Comment:
since we already had a policies API integration test in cypress, I decided to port the policy behavior tests in https://github.com/chef/automate/blob/master/inspec/a2-api-integration/controls/authz_api.rb here